server/helpers/utils.js

```diff
@@ -1,9 +1,114 @@
 import bcrypt from 'bcryptjs';
+import validator from 'validatorjs';
+import jwt from 'jsonwebtoken';
+import sendGrid from '@sendgrid/mail';
 
 /**
  * @param {string} password
  * @return {string} hash
  */
-const hashPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync(10));
+export const hashPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync(10));
 
-export default hashPassword;
+/**
+ * @param {object} inputObject
+ * @return {object} sanitized data
+ */
+export const trimAndConvertToLowercase = (inputObject) => {
+  const sanitizedData = {};
+  Object.entries(inputObject).forEach((element) => {
+    const key = element[0];
+    const value = element[1];
+    sanitizedData[key] = value.replace(/\s/g, '').toLowerCase();
+  });
+  return sanitizedData;
+};
+
+/**
+ * @param {object} data
+ * @param {object} rules
+ * @param {object} res
+ * @return {object} response
+ */
+export const checkValidation = (data, rules) => {
+  const validation = new validator(data, rules);
+
+  return validation.passes() ? true : validation.errors.all();
+};
+
+/**
+ * @param {object} res
+ * @param {integer} statusCode
+ * @param {string} message
+ * @param {object} data
+ * @param {boolean} success
+ * @return {object} response
+ */
+export const displayMessage = (
+  res, statusCode, message, data, success = true
+) => {
+  const payload = success === true ? 'data' : 'error';
+
+  return res.status(statusCode).json({
+    status: success === true ? 'success' : 'error',
+    message,
+    [payload]: data
+  });
+};
+
+/**
+ * @param {string} id
+ * @param {string} isVerified
+ * @param {string} tokenExpiryDate
+ * @param {string} secret
+ * @param {string} email
+ * @return {string} token
+ */
+export const tokenGenerator = (id, isVerified, tokenExpiryDate = '1h', secret = 'secret', email) => {
```

Comment:
We could reduce the number of params that are passed to this function, considering the `payload` is an object, see suggestion below 👇 

```suggestion
export const tokenGenerator = (payload, tokenExpiryDate = '1h', secret = 'secret') => {
```